sklearn/metrics/pairwise.py

```diff
@@ -1109,6 +1188,45 @@ def rbf_kernel(X, Y=None, gamma=None):
     return K
 
 
+def choi_kernel(X: np.ndarray, Y: np.ndarray=None, nan_stdev_multiplier: float = 3., gamma=None):
+    """
+    First, compute the standard deviation of each column. Then, compute the distance between X and Y as follows:
```

Comment:
Is this a handling nan version sort of Gaussian Kernel?

Reply:
Yes